bears/java/CheckstyleBear.py

```diff
@@ -19,12 +20,21 @@ class CheckstyleBear:
     """
     Check Java code for possible style, semantic and design issues.
 
+    By default the Google Java Style coding conventions are used.
+
     For more information, consult
     <http://checkstyle.sourceforge.net/checks.html>.
     """
 
     LANGUAGES = "Java"
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-        return '-jar', checkstyle_jar_file, '-c', google_checks, filename
+    def create_arguments(filename, file, config_file,
+                         checkstyle_config: path=None):
```

Comment:
why not set it to `google_checks` by default instead of the unneeded if else below?